src/chatlog/chatmessage.cpp

```diff
@@ -175,6 +181,72 @@ void ChatMessage::hideDate()
         c->hide();
 }
 
+QString ChatMessage::detectMarkdown(const QString &str)
+{
+    QString out;
+
+    // Create regex for certain markdown syntax
+    QRegExp exp("(\\*\\*)([^\\*\\*]{2,})(\\*\\*)"   // Bold    **text**
+                "|(\\*)([^\\*]{2,})(\\*)"           // Bold *text*
+                "|(\\_)([^\\_]{2,})(\\_)"           // Italics _text_
+                "|(\\_\\_)([^\\_\\_]{2,})(\\_\\_)"  // Italics __text__
```

Comment:
Hmm, github-flavoured markdown parses `*text*` as italics - *text*`

Additionally, `__text__` is being parsed as bold - __text__